packages/lodestar-validator/src/api/abstract.ts

```diff
@@ -118,4 +124,12 @@ export abstract class AbstractApiClient
     }
   };
 
+  /**
+   * Returns milis till next slot
+   */
+  private getDiffTillNextSlot(): number {
+    const diffInSeconds = Math.floor((Date.now() / 1000) - this.genesisTime);
```

Comment:
this function will likely return seconds to next slot * 1000. Should we improve it to calculate more precisely to return milli seconds to next slot?

Reply:
I find it hard to implement simple algo for this. That said,I don't think we care about milis there.